src/components/Link.tsx

```diff
@@ -89,6 +89,14 @@ export const Link = (props: LinkProps) => {
   } = props;
   const { locale } = useIntl();
   const isExternal = to && to.includes('http');
+  // Check for ethereum.org root domain links
+  const ETHEREUM_DOT_ORG = 'ethereum.org';
+  const isEthereumDotOrg =
```

Comment:
I think here we can just check if `to.includes('https://ethereum.org')`. Let's update any links that contain `http` or `www` - we shouldn't encourage that type of linking.